SingularityService/src/main/java/com/hubspot/singularity/config/MesosConfiguration.java

```diff
@@ -22,6 +22,10 @@
   private int defaultMemory = 64;
 
   private int defaultDisk = 0;
+  // ^ This refers to SingularityDeploy-scoped resources.
+
+  // v This refers to Mesos TaskInfo-scoped resources. Changing this doesn't require a redeploy of the world.
+  private int mesosTaskBuilderDiskAllocationMb = 1;
```

Comment:
Trying to think of a clearer name for this, but nothing coming to mind at the moment